addons/web/static/src/legacy/scss/icons.scss

```diff
@@ -0,0 +1,72 @@
+// = Icons
```

Comment:
@stefanorigano Why in legacy? Do you plan to get rid of after migration?

Reply:
`/core`? @aab-odoo ?

Reply:
/core if its meant to be used outside the webclient (so not only in /web), webclient/ otherwise

Reply:
Ok, thanks.
Let's go for `/webclient`... we'll move it in `core` if/when these icons will be used  ouside  `/web`.